tests/jballerina-unit-test/src/test/resources/test-src/statements/matchstmt/const-pattern.bal

```diff
@@ -405,6 +405,23 @@ function testConstPattern12() {
     assertEquals("OTHER", constPattern12(3));
 }
 
+client class Client {
+    remote function post(string x, string y) returns anydata {
+        return y;
+    }
+}
+
+Client clientEndpoint = new ();
+public function testConstPattern13() {
+    string animal = "Mouse";
+    match animal {
+        "Mouse" => {
+            var res = clientEndpoint->post("/post", "POST: Hello World");
+            assertEquals("POST: Hello World", res);
+        }
```

Comment:
We should probably have a wildcard match pattern here and panic if it reaches that block, just for completeness.

Reply:
Fixed.